src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/resource_providers/__init__.py

```diff
@@ -73,10 +73,11 @@ def _update_private_endpoint_connection_status(self, cmd, resource_group_name,
             query_counts -= 1
             private_endpoint_connection = self.show_private_endpoint_connection(cmd,
                                                                                 resource_group_name,
-                                                                                service_name,
+                                                                                resource_name,
                                                                                 private_endpoint_connection_name)
-            if private_endpoint_connection['properties'].get('provisioningState'):
-                if private_endpoint_connection['properties']['provisioningState'] in ["Succeeded", "Ready"]:
+            if private_endpoint_connection['properties'].get('provisioningState', None) in ["Succeeded", "Ready"]:
```

Comment:
This is the code change. @haroldrandom

Reply:
cosmosdb has a quite weird behavior so I have to check more values. It won't have bad effect on other services.